### PR TITLE
New Member

### PR DESCRIPTION
**Pull request by akannshu**

### CODE REVIEW

File: Members.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ Welcome to BVP MOSC
 - [arush15june](https://github.com/arush15june)
 - IT E (2017-21)
 ---
+#### AKanshu Raj
```
Comment (jainaman224) [inline Members.md]: I think i should be `Akanshu` instead of `AKanshu`.

**Review by jainaman224 — CHANGES_REQUESTED** [review]

**Comment by akannshu** [comment]
its Akanshu only

**Comment by jainaman224** [comment]
Change ~AKanshu~ to **Akanshu**.
Look closely at `k`.

**Comment by akannshu** [comment]
done

**Review by jainaman224 — APPROVED** [review]

**Pull request merged**